api.go

```diff
@@ -32,9 +31,15 @@ const (
 	API_ERR_INTERNAL
 )
 
+type deploymentsResult struct {
+	deployments []DataDeployment
+	err         error
+}
+
 var (
-	deploymentsChanged = make(chan string)
-	addSubscriber      = make(chan chan string)
+	deploymentsChanged = make(chan interface{})
```

Comment:
Yes, we can try and test the best buffer size in a later version. Just a potential improvement.